airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/PostgresCli.java

```diff
@@ -0,0 +1,55 @@
+package io.airbyte.integrations.destination.postgres;
+
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.db.jdbc.JdbcDatabase;
+import io.airbyte.db.jdbc.JdbcUtils;
+import io.airbyte.integrations.base.ssh.SshTunnel;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang3.tuple.Pair;
+
+public class PostgresCli {
+
+  public static void main(final String[] args) throws Exception {
+    if (args.length < 3 || args.length % 2 != 1) {
+      System.out.println("Usage: java -jar whatever.jar <postgres-config> <schema1> <table1> [<schema2> <table2> ...]");
+      System.exit(1);
+    }
+
+    final String configStr = args[0];
+    final List<Pair<String, String>> tables = new ArrayList<>();
+    for (int i = 1; i < args.length; i += 2){
+      tables.add(Pair.of(args[i], args[i + 1]));
+    }
```

Comment:
again just gonna state the logic here
this loops through the args, skipping the config arg and shouldn't have an index out of bounds error because
a) we've insured the list is odd
b) you're incrementing by 2 so the last iteration is length - 2 or second to last (breaking out on length or last index + 1)